SymbolCollisionTest/Podfile

```diff
@@ -60,12 +61,13 @@ target 'SymbolCollisionTest' do
     pod 'GooglePlacePicker'
     pod 'GooglePlaces'
     pod 'GooglePlusUtilities'
+    #TODO: Uncomment next two when GTMAppAuth updates to GTMSessionFetcher 2.x
```

Comment:
again here if you think it'd be helpful.

Reply:
Actually already resolved. Deleted comment.